jupyter-no-output/jupyter-no-output.rs

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env rust-script
+//!
+//! ```cargo
+//! [dependencies]
+//! anyhow = "1.0.*"
+//! serde_json = "1.0.*"
+//! ```
+
+use std::process::{Command, ExitCode};
+
+use anyhow::{self as A, bail};
```

Comment:
Why use the abbreviation `A`? Seems to read clearer if the `anyhow` is kept as is.

Reply:
Thanks, I'll change it.